src/elfcore.cc

```diff
@@ -662,13 +662,13 @@ static int WriteThreadRegs(void *handle, ssize_t (*writer)(void *, const void *,
  * to also return the address of VDSO Elf header, if AT_SYSINFO_EHDR
  * is present.
  */
-static void CountAUXV(size_t *pnum_auxv, size_t *pvdso_ehdr, char *proc_path, int proc_file_ind, bool ext_proc) {
+static void CountAUXV(size_t *pnum_auxv, size_t *pvdso_ehdr, char *proc_path, int proc_file_ind) {
```

Comment:
I think you missed the intent of my previous comment.

I meant that you should only pass an "ext_pid" value around, and construct the full proc path in OpenProcFile() each time. That way you're consistent in the additional argument you're adding.